### PR TITLE
Add data migration for dos entries in lots table

### PR DESCRIPTION
**Pull request by carolinegreen**
Add data migration so can be certain lots are consistant through all the enivironments.

This has been rebased on top of https://github.com/alphagov/digitalmarketplace-api/pull/295 because both migrations need to go in together.

### CODE REVIEW

**Comment by quis** [comment]
This should probably go in after https://github.com/alphagov/digitalmarketplace-api/pull/295, with the lot-framework relationship defined.

**Comment by quis** [comment]
Tested locally and does the job. @carolinegreen @robyoung If you're happy with mine and @TheDoubleK’s earlier commits then this is :+1:.

File: tests/app/views/test_suppliers.py [hunk 1]
```diff
@@ -1257,9 +1255,7 @@ def setup(self):
         self.setup_dummy_suppliers(1)
 
         with self.app.app_context():
-            framework_enum_vals = db.session.execute("SELECT enum_range(NULL::framework_enum);").first()[0]
-            if 'dos' not in str(framework_enum_vals):
-                self.bootstrap_dos()
```
Comment (pcraig3) [inline tests/app/views/test_suppliers.py]: This is great, getting rid of this.

**Comment by pcraig3** [comment]
I think after removing the enum check in those 3 tests, you can also remove the [`bootstrap_dos()`](https://github.com/alphagov/digitalmarketplace-api/blob/b597eee092f2ede31173ffffb5cd8db9e7f3c2b8/tests/app/helpers.py#L184) method entirely now.

**Comment by TheDoubleK** [comment]
Yep we should definitely be able to :fire: :fire: :fire: `bootstrap_dos()` from the tests everywhere.

**Pull request merged**